File: Java/SivitasAkademik.java
class SivitasAkademik extends Human {
    private String asal_universitas;
    private String email_edu;

    SivitasAkademik() {
    }

    SivitasAkademik(String asal_universitas, String email_edu) {
        this.asal_universitas = asal_universitas;
        this.email_edu = email_edu;
    }

    public void setasal_universitas(String asal_universitas) {
        this.asal_universitas = asal_universitas;
    }

    public void setemail_edu(String email_edu) {
        this.email_edu = email_edu;
    }

    public String getasal_universitas() {
        return this.asal_universitas;
    }

    public String getemail_edu() {
        return this.email_edu;
    }
}

File: Python/SivitasAkademik.py
from Human import Human


class SivitasAkademik(Human):
    __asal_universitas = ""
    __email_edu = ""

    def __init__(self, asal_universitas="", email_edu=""):
        self.__asal_universitas = asal_universitas
        self.__email_edu = email_edu

    def getemail_edu(self):
        return self.__email_edu

    def getasal_universitas(self):
        return self.__asal_universitas

    def setasal_universitas(self, asal_universitas):
        self.__asal_universitas = asal_universitas

    def setemail_edu(self, email_edu):
        self.__email_edu = email_edu
